### PR TITLE
Test updated projects imgreg views

### PR DESCRIPTION
**Pull request by jdemon519**
{{pytest: cfme/tests/containers/test_file_name.py -v --use-provider container }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 0945bf8463acba07d2b63e9680b4d4fe31855484:

`cfme/tests/containers/test_imgreg_proj_views.py`:
- :bangbang: Line 35:26: **E901** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .
*CFME QE Bot*

File: cfme/tests/containers/test_containers_properties.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        # verify that mgmt pods exist in ui listed pods
+        assert set(ui_pods).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_pods:
+        obj = Pod(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Tautology

File: cfme/tests/containers/test_containers_properties.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        # verify that mgmt routes exist in ui listed routes
+        assert set(ui_routes).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_routes:
+        obj = Route(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Tautology

File: cfme/tests/containers/test_containers_properties.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        # verify that mgmt projects exist in ui listed projects
+        assert set(ui_projects).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_projects:
+        obj = Project(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_properties.py]: Tautology

File: cfme/tests/containers/test_containers_services.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        # verify that mgmt services exist in ui listed services
+        assert set(ui_services).issubset(
+            [obj.name for obj in mgmt_objs]), 'Missing objects'
+
+    for name in ui_services:
+        obj = Service(name, provider)
+        field_content = getattr(obj.summary.properties, rel).text_value
+
+        if field_content:
+            assert len(field_content) != 0
```
Comment (mfalesni) [inline cfme/tests/containers/test_containers_services.py]: Tautology

**Comment by dajoRH** [comment]
Lint report for commit 8ecf741ca6b18d95beffe89e911a889efea2ba5b:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/containers/test_provider_crud.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import uuid
```
Comment (jkrocil) [inline cfme/tests/containers/test_provider_crud.py]: This shouldnt be here. Its unrelated to the rest of the PR. You are just adding a file that was removed some time ago by mistake. Please make a separate PR fixing this specific issue of missing CRUD tests.

File: cfme/tests/containers/test_containers_properties.py (hunk 1 above)
Comment (psav) [inline cfme/tests/containers/test_containers_properties.py]: ```assert len(field_content)```

will do the job of

```if field_content:
    assert len(field_content) != 0
```

**Comment by psav** [comment]
#3328 is a newer version of this? If so please close one of them off so we don't double review things
